### DISCUSSION TITLE
ValueError("Reference at 'refs/heads/master' does not exist") when submitting datapackage

### DISCUSSION THREAD

**Discussion by sherxie** [nih-cfde/published-documentation#270] [Q&A] [answered]
Hi, I am trying to submit a datapackage but have been running into this error each time:

![Screen Shot 2021-09-29 at 4 02 12 PM](https://user-images.githubusercontent.com/39966933/135340226-43664d41-f4ba-48a1-86a0-fa7f982c36e7.png)

I've tried logging out and back in as well as updating the cfde-submit package, but it seems I already have the latest version installed. The datapackage itself is local to my computer, not a Git repository. 

Any assistance would be appreciated! Thank you.

**Comment by ACharbonneau** [comment]
Checking into this now

**Reply by DavidMichaelKelly** [reply]
It looks like cfde-submit thinks your directory as a git repo. Is there a .git directory in your draft_submission folder? Can you try running it with git disabled? cfde-submit run --ignore-git draft_submission_092221

**Reply by sherxie** [reply]
Hmm, there's no .git directory so I'm not sure why that happened. The submission worked with the --ignore-git option though, thank you! 

Unfortunately, the ingest failed for the following reason (I can open this as a separate issue, if needed):

![Screen Shot 2021-09-29 at 5 55 35 PM](https://user-images.githubusercontent.com/39966933/135354212-9b961768-2340-4e4a-8f42-40b44a0c2778.png)

The datapackage passed the validation step because I'm using the most recent C2M2 schema, but should I not be including the new tables in the submission yet?

**Reply by ACharbonneau** [reply]
Those are in the current master schema. The portal should be accepting this JSON as of September 1st: https://osf.io/vzgx9/  

I'm checking into it now

**Comment by ACharbonneau** [comment, marked as answer]
This was a bug on our production server that we pushed a fix for in October 2021.
